docs/updating/4_update_2.2.md

```diff
@@ -112,7 +112,7 @@ After the migration is finished, you need to clear the cache.
 
 #### Migrate layouts
 
-The `ez_block:renderBlockAction` controller used in layout templates has been replaced by `EzPlatformPageFieldTypeBundle:Block:render`. This controller has two additional parameters, `locationId` and `languageCode`. Only `languageCode` is required.
+The `ez_block::renderBlockAction` controller used in layout templates has been replaced by `EzPlatformPageFieldTypeBundle:Block:render`. This controller has two additional parameters, `locationId` and `languageCode`. Only `languageCode` is required.
```

Comment:
It's possible that the second mentioned controller needs to be adjusted as well, and all other occurences if there are any.